admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/domain/model/UserQuery.kt

```diff
@@ -0,0 +1,14 @@
+package net.thechance.mena.admin_panel.domain.model
+
+data class UserQuery(
+    val searchInput: String?,
+    val sortType: String?,
+    val sortDirection: SortDirection?,
+    val page: Int = 0,
+    val size: Int = 20
+
+)
+enum class SortDirection {
+    ASC,
+    DESC
+}
```

Comment:
```suggestion
enum class SortDirection(val value: String) {
    ASC("asc"),
    DESC("desc")
}
```

I suggest to give it value and when you use it you don't need to make `when` statement just call .value